### DISCUSSION TITLE
Problems with the way we use data and GIS

### DISCUSSION THREAD

**Discussion by alizezila** [albertkun/21S-ASIAAM-191A#21] [Past Thinking Caps]
Hey all! 

I mentioned this in class, but I think a major problem for data, GIS, and most data visualization is addressing bias and how it may present itself in our data. As professionals working with data and tasked to present it to the world, it is so so important that we acknowledge biases, prejudices, assumptions etc. about communities that may be included in the data. I think we tend to take data for granted, that it is completely objective and truthful and cemented. But in reality, there are real people with real biases behind much of the data and coding that may affect how others perceive it. Like in the first couple pages of our textbook, the authors gave us various examples of how the same exact data can be visually represented differently and convey different messages. It is important as professionals we try to step outside of our work and understand how others will understand it all.

**Comment by ghost** [comment]
I completely agree with everything you mention above! I think it is very important to be aware of how much bias (especially unconscious bias) affects our everyday lives, education, work, etc. Data is an amazing tool to have available at our fingertips that it can be easy to forget that while it may just seem like numbers and words, it represents information about real people. These real people have to live with the consequences produced by the misuse or misrepresentation of data. This is especially true for data visualized using GIS because maps can be very convincing and easily distort data in ways that seriously impact different communities of people. I think acknowledgment and awareness of bias is a good first step to attempt to instill in the minds of individuals using widely accessible data.
